marl_eval/json_tools/pull_neptune_data.py

```diff
@@ -0,0 +1,71 @@
+# python3
+# Copyright 2022 InstaDeep Ltd. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+import zipfile
+from typing import List
+
+import neptune
+from colorama import Fore, Style
+from tqdm import tqdm
+
+
+def pull_neptune_data(
+    project_name: str,
+    tag: List,
+    store_directory: str = "./downloaded_json_data",
+    neptune_data_directory: str = "metrics",
+) -> None:
+    """Pulls the experiments data from Neptune to local directory.
+
+    Args:
+        project_name (str): Name of the Neptune project.
+        tag (List): List of tags.
+        store_directory (str, optional): Directory to store the data.
+        neptune_data_directory (str): Directory in Neptune where the data is stored.
```

Comment:
Just changing this to `neptune_data_key`. Feel free to reject. But directory feels like it is a local directory we are referencing when we are actually referring to a key in the run where the json data is stored. 
```suggestion
        neptune_data_key (str): Key in the neptune run where the json data is stored. 
```